### PR TITLE
Updated MySQLProperties.tsx

### PR DESCRIPTION
**Pull request by cubsg**
Fixes https://github.com/Canner/WrenAI/issues/712

I've added changes to the MySQLProperties.tsx file. I edited line 76, replacing the password placeholder from "input password" to "************".

### CODE REVIEW

**Comment by jameskim0987** [comment]
I just came across the related issue ticket. It's probably a good idea to add "Fixes #712" in the description so the issue is linked.

File: wren-ui/src/components/pages/setup/dataSources/MySQLProperties.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default function MySQLProperties(props: Props) {
           },
         ]}
       >
-        <Input.Password placeholder="input password" />
+        <Input.Password placeholder="************" />
```
Comment (fredalai) [inline wren-ui/src/components/pages/setup/dataSources/MySQLProperties.tsx:76]: The `<MySQLProperties/>` component is used in the onboarding process and the data source settings. 
This change will be different from what is expected on the onboarding process side, we still want to keep the placeholder as the **input password** string.

**Review by fredalai — COMMENTED** [review]
Hi @cubsg,
Thank you for your continued efforts to resolve this issue. However, I noticed that the current solution introduces a new bug and deviates from the solution I provided previously.

To help guide you in the right direction, I have clarified the expected behavior and solution:

#### Updated Expected Behavior and Solution:
 - After completing the onboarding process, when the user views the Data Source Settings, the password field should display `************` instead of being empty.
 - This fix needs to be applied to data sources such as ClickHouse, SQL Server, MySQL, PostgreSQL, and Trino. Currently, only the PostgreSQL data source behaves as expected.

Please feel free to reach out if you need further clarification or assistance. Thank you for your understanding and continued efforts on this issue!

**Comment by fredalai** [comment]
Thanks @jameskim0987

**Comment by fredalai** [comment]
Hi @cubsg,

I hope you're doing well! I noticed that this PR hasn't been updated in a while, and with our Wren AI Hacktoberfest ending in just two weeks, now is a great time to push forward. 
If you're facing any challenges or need assistance, I'd be happy to help. Please let me know if there's anything I can do to assist in moving this PR forward.

Thank you for your contribution, and let's make the most of this Hacktoberfest!

Best regards,
Freda

**Comment by fredalai** [comment]
Hi @cubsg,

I hope you're doing well! I wanted to follow up on the progress of this PR, as we haven’t seen any updates in over a month.

Unfortunately, as per our Hacktoberfest guidelines, we need contributors to provide updates at least every 5 days to allow others a chance to work on the issue. If we don't hear back soon, we will need to close this PR and reassign the issue to another contributor to keep things moving forward.

I truly appreciate the effort you've put into this so far. If you're still interested in working on it, please let us know with an update. 
Thank you again for your contribution, and we hope to see you participate in future events!

Best regards,
Freda

**Comment by fredalai** [comment]
Follow the Hacktoberfest guidelines and reassign the issue task.

**Pull request closed without merging**